CHANGELOG.md

```diff
@@ -25,14 +24,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix syntax lexer guessing.
 - Fixed Pretty measure not respecting expand_all https://github.com/Textualize/rich/issues/1998
 - Collapsed definitions for single-character spinners, to save memory and reduce import time.
-- Fix print_json indent type in __init__.py
+- Fix print_json indent type in **init**.py
```

Comment:
Not exactly an issue but just curious - is this some markdown formatter thing that caused this?

Reply:
Yeah, I think that may have been vscode. I need to put it inside backticks.